roles/openshift_registry/tasks/main.yml

```diff
@@ -1,11 +1,14 @@
 ---
-- set_fact: _oreg_images="--images={{ oreg_url|quote }}"
-  when: oreg_url is defined
+
+- set_fact: _oreg_images="--images='{{ openshift.master.registry_url }}'"
+
+- set_fact: _oreg_selector="--selector='{{ openshift.master.registry_selector }}'"
 
 - name: Deploy OpenShift Registry
   command: >
     {{ openshift.common.admin_binary }} registry
-    --create
-    --credentials={{ openshift_master_config_dir }}/openshift-registry.kubeconfig {{ _oreg_images|default() }}
+    --create --service-account=registry
+    {{ _oreg_selector }} --replicas=0
```

Comment:
Drop the --replicas=0 now that we've added the selector defaults? We should probably add more options for the registry due to the storage requirements but we can do that in a follow up.